src/components/button/button.jsx

```diff
@@ -24,34 +25,20 @@ module.exports = React.createClass({
     id: React.PropTypes.string
   },
 
-  render: function() {
-    return this.props.href ? this.renderAnchor() : this.renderButton();
-  },
-
-  renderAnchor: function() {
-    const props = _.omit(this.props, 'children');
+  _getProps() {
+    const props = _.omit(this.props, ['data', 'size', 'purpose']);
```

Comment:
This is something I noticed yesterday so thought I'd kill 2 birds...

https://holidayextras.jira.com/browse/WEB-15447

* no actual birds were killed in the making of this PR